util/regexp-assemble/data/932105.data

```diff
@@ -94,6 +94,7 @@ $\(\(
   ##!> cmdline unix
 
 ##! words starting with a-n are in 932100.data
```

Comment:
Not true... `m` and `n` are in this file. Maybe we need to move some over here or vice-versa.

Reply:
The comment was wrong. a-l are in that file. Updated comment.